pkg/scheduler/backend/queue/scheduling_queue_test.go

```diff
@@ -104,6 +104,10 @@ var (
 	}
 )
 
+func init() {
+	metrics.Register()
```

Comment:
Actually, I'm wondering why we don't just do `metrics.Register` in `init()` of metrics package.
But, anyway, I don't prefer to include such change unrelated to the enhancement in this PR, so just do `init()` here. We can follow up in another PR.

Reply:
Maybe a related pr: https://github.com/kubernetes/kubernetes/pull/127612/files

We should definitely find a way to handle the `metrics.Register`, agree should not be included in this pr, but maybe in #127612

Reply:
This was intentional.
https://github.com/kubernetes/kubernetes/pull/126762

Please check if the conclusions still apply.

Reply:
Got it, looks like the reason is still appropriate.
Like discussed in https://github.com/kubernetes/kubernetes/pull/127612 and I did here, probably we can just do `init()` on each test file.